backend/requirements.txt

```diff
@@ -1,5 +1,6 @@
 pymupdf==1.22.3
-langchain==0.0.247
+langchain==0.0.281
+litellm==0.1.531
```

Comment:
Do we still need this import ?

Reply:
I think so :) langchain is just a wrapper. It doesn't come with the packages.

Reply:
Sounds good